Rotativa/WkhtmlDriver.cs

```diff
@@ -15,7 +17,33 @@ public abstract class WkhtmlDriver
         /// <param name="html">String containing HTML code that should be converted to PDF.</param>
         /// <param name="wkhtmlExe"></param>
         /// <returns>PDF as byte array.</returns>
-        protected static byte[] Convert(string wkhtmlPath, string switches, string html, string wkhtmlExe)
+        protected static byte[] Convert(string wkhtmlPath, string switches, string html, string wkhtmlExe, int? timeout = null)
+        {
+            if (!timeout.HasValue)
+            {
+                return ConvertExecute(wkhtmlPath, ref switches, ref html, wkhtmlExe);
+            }
+            else
+            {
+                var cancellationTokenSource = new CancellationTokenSource();
+                var task = Task.Run(() => ConvertExecute(wkhtmlPath, ref switches, ref html, wkhtmlExe, cancellationTokenSource.Token));
```

Comment:
this is not as I could imagine best, but it works. 

I've tried to do it by calling 
`proc.WaitForExit(milliseconds: timeeout)`
but this requires that **output** of process is **asynchronous**. 

I've tried to do it also by implementing events:
```
proc.OutputDataReceived += (sender, outputLine) => ...
proc.ErrorDataReceived += (sender, errorLine) => ...
```
but I failed to convert **outputLine.Data** to bytes[]. Thats cuz it is String, not Stream (as in proc.StandardOutput.BaseStream).

I've tried provide some encoding to process output and convert it, but any way I tried to do it, the resulted PDF was broken. 

Finally I gave up and it that way